applications/finetune-quora-embeddings/optimize.json

```diff
@@ -0,0 +1,372 @@
+
```

Comment:
can you make a csv of this

Reply:
Converted to csv file of `prev_results.json`